### PR TITLE
Added devise and css design changes

### PR DESCRIPTION
**Pull request by danielsteenberg**

### CODE REVIEW

File: spec/models/model_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
```
Comment (ryannealmes) [inline spec/models/model_spec.rb]: This file looks wrong. A model that is called model?

**Comment by ryannealmes** [comment]
In future I am not going to look at any PR that doesn't have a description of what changes are included in the PR. It makes it really difficult. Also I have repeatedly commented on formatting. Please can you check spacing and whitespace and readability in general before committing you code. Another point is to keep your PRs small. I know this devise change requires a bit of code so it's fine in this case, but I am just raising it as something to keep in the back of your mind.

**Comment by danielsteenberg** [comment]
Hi Ryan, 

Changed the Model to user lol, was testing with that but i know i shouldn't have that as a name lol, so i made a user one for now just for login purposes. There will be an admin one that will follow at a later stage. Will sort out all layout to two finger spacing for the next PR. 

Thanks a mil

**Comment by ryannealmes** [comment]
Cool, there are a couple of other things I would change, but happy for you to merge.

**Pull request merged**